firecracker-control/cmd/containerd/README.md

```diff
@@ -0,0 +1,4 @@
+# README
+
+This is custom `containerd` binary which embeds `firecracker-control` GRPC plugin (instead of using it as golang
```

Comment:
nit: "a custom `containerd` binary", "embeds the `firecracker-control` GRPC plugin", "instead of using it as a Go plugin", "simplify the development process"